Geometry/TrackerCommonData/data/2016/FlatMinus05Percent/trackermaterial.xml

```diff
@@ -500,5 +500,78 @@
         <rMaterial name="materials:C6F14" />
       </MaterialFraction>
     </CompositeMaterial>
+
+    <!-- Phase1 BPIX micro-twisted pairs signal cables material as defined by W. Bertl -->
+    <CompositeMaterial name="micro_twisted_signal_cable" density="0.95*4.09*g/cm3" symbol=" " method="mixture by weight">
+      <MaterialFraction fraction="0.476683">
+        <rMaterial name="trackermaterial:T_Aluminium"/>
+      </MaterialFraction>
+      <MaterialFraction fraction="0.274139">
+        <rMaterial name="trackermaterial:T_Copper"/>
+      </MaterialFraction>
+      <MaterialFraction fraction="0.1494102">
+        <rMaterial name="materials:Carbon"/>
+      </MaterialFraction>
+      <MaterialFraction fraction="0.02006116">
+        <rMaterial name="materials:Hydrogen"/>
+      </MaterialFraction>
+      <MaterialFraction fraction="0.07960754">
+        <rMaterial name="materials:Oxygen"/>
+      </MaterialFraction>
+    </CompositeMaterial>
+    <!-- Phase1 BPIX micro-twisted pairs power cables material as defined by W. Bertl -->
+    <CompositeMaterial name="micro_twisted_power_cable" density="0.95*3.42*g/cm3" symbol=" " method="mixture by weight">
+      <MaterialFraction fraction="0.751011">
+        <rMaterial name="trackermaterial:T_Aluminium"/>
+      </MaterialFraction>
+      <MaterialFraction fraction="0.138483">
+        <rMaterial name="trackermaterial:T_Copper"/>
+      </MaterialFraction>
+      <MaterialFraction fraction="0.066287">
+        <rMaterial name="materials:Carbon"/>
+      </MaterialFraction>
+      <MaterialFraction fraction="0.0089008">
+        <rMaterial name="materials:Hydrogen"/>
+      </MaterialFraction>
+      <MaterialFraction fraction="0.0353182">
+        <rMaterial name="materials:Oxygen"/>
+      </MaterialFraction>
+    </CompositeMaterial>
+  
+    <!--OUTDATED Phase1 BPIX micro-twisted pairs boundle material as defined by W. Bertl, effective density defined to matcht the measured weigth (0.0071+0.0208 g/cm) in a single 1.5mm diam. boundle-->
+    <CompositeMaterial name="micro_twisted_boundle" density="0.95*1.585*g/cm3" symbol=" " method="mixture by weight">
+      <MaterialFraction fraction="0.7455">
+        <rMaterial name="trackermaterial:micro_twisted_power_cable"/>
+      </MaterialFraction>
+      <MaterialFraction fraction="0.2545">
+        <rMaterial name="trackermaterial:micro_twisted_signal_cable"/>
+      </MaterialFraction>
+    </CompositeMaterial>
+  
+    <!--Phase1 BPIX micro-twisted pairs boundle material as defined by L. Caminada-->
+    <CompositeMaterial name="micro_twisted_power_cable" density="0.95*3.639*g/cm3" symbol=" " method="mixture by weight">
```

Comment:
@bsunanda, @ianna There are two "micro_twisted_power_cable" composite materials defined here. Aren't duplicate names an error? What effect does this duplication have? Does anyone in CMS know?